### PR TITLE
Add pagination to namespace list view

### PR DESCRIPTION
**Pull request by arnav28**
Added client side pagination for namespaces list view. Default is 15 per page.
Todo: Unit testing

Before (2000 namespaces):
https://user-images.githubusercontent.com/2932708/143089534-4b3785f2-35b0-43bf-a8ff-b371d0967f82.mov

After:
https://user-images.githubusercontent.com/2932708/143089596-a17f1684-3ba3-49aa-85df-b4f4a2990f35.mov

### CODE REVIEW

File: ui/app/routes/vault/cluster/access/namespaces/index.js [hunk 1]
```diff
@@ -3,21 +3,74 @@ import Route from '@ember/routing/route';
...
-        })
-      : null;
+        });
+    }
+    return null;
+  },
+
+  setupController(controller, model) {
+    const has404 = this.has404;
+    controller.set('hasModel', true);
```
[the remark this answers is not in the document]
Comment (arnav28) [reply ui/app/routes/vault/cluster/access/namespaces/index.js]: Yup, updated.

**Review by hashishaw — APPROVED** [review]
Thank you!

File: ui/tests/acceptance/access/namespaces/index-test.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { currentRouteName } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import page from 'vault/tests/pages/access/namespaces/index';
+import authPage from 'vault/tests/pages/auth';
+import logout from 'vault/tests/pages/logout';
+
+module('Acceptance | /access/namespaces', function(hooks) {
```
Comment (hashishaw) [inline ui/tests/acceptance/access/namespaces/index-test.js:9]: Need to add `Enterprise` to this module name, since namespaces are enterprise only

Comment (arnav28) [reply ui/tests/acceptance/access/namespaces/index-test.js]: You saved my day...Thanks!! 😅

**Pull request merged**